<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Baiboly 1.0</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="actionCouper"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuPr_f_rences">
    <property name="title">
     <string>Préférences</string>
    </property>
    <widget class="QMenu" name="menuLangue_par_d_faut">
     <property name="title">
      <string>Langue par défaut</string>
     </property>
     <addaction name="actionMalgache"/>
     <addaction name="actionFrancais"/>
     <addaction name="actionAnglais"/>
    </widget>
    <addaction name="menuLangue_par_d_faut"/>
    <addaction name="stripNumberAction"/>
   </widget>
   <addaction name="menuEdition"/>
   <addaction name="menuPr_f_rences"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCopier">
   <property name="text">
    <string>Copier</string>
   </property>
  </action>
  <action name="actionColler">
   <property name="text">
    <string>Coller</string>
   </property>
  </action>
  <action name="actionS_lectionner_tout">
   <property name="text">
    <string>Sélectionner tout</string>
   </property>
  </action>
  <action name="actionCouper">
   <property name="text">
    <string>Imprimer</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionMalgache">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Malgache</string>
   </property>
  </action>
  <action name="actionFrancais">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Français</string>
   </property>
  </action>
  <action name="actionAnglais">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Anglais</string>
   </property>
  </action>
  <action name="stripNumberAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Afficher les numérotations</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
